code/game/machinery/buttons.dm

```diff
@@ -32,6 +32,7 @@
 
 /obj/machinery/button/attack_hand(mob/living/user)
 	if(..()) return 1
+	user.visible_message("<span class='notice'>[user] hits the [src.name] button.</span>")

```

Comment:
```suggestion
	user.visible_message("<span class='notice'>[user] hits [src] button.</span>")
```

Reply:
Then it doesn't add thes.
![image](https://user-images.githubusercontent.com/17379794/62938625-0c645b80-bdd8-11e9-8718-d797bf55ca61.png)
Looks pretty ugly.

Reply:
```suggestion
	user.visible_message("<span class='notice'>[user] hits \the [src] button.</span>")
```